src/uwtools/cli.py

```diff
@@ -52,9 +52,9 @@ def main() -> None:
         setup_logging(quiet=True)
         args, checks = _parse_args(sys.argv[1:])
         args[STR.action] = args.get(STR.action, args[STR.mode])
-        for check in checks[args[STR.mode]][args[STR.action]]:
+        for check in checks[args[STR.mode]].get(args[STR.action], []):
             check(args)
-        setup_logging(quiet=args[STR.quiet], verbose=args[STR.verbose])
+        setup_logging(quiet=args.get(STR.quiet, False), verbose=args.get(STR.verbose, False))
```

Comment:
For drivers, the argument value called `action` here would be the driver task name (`provisioned_rundir`, `run`, `validate`, etc.). (For other modes it'd be e.g. `realize` in `config realize`, `render` in `template render`, etc.) But when `--show-schema` is specified, a task (action) wouldn't be expected: We'd want to be able to say `uw upp --show-schema` without specifying any particular task. So, in the upper diff, the case where no `action` is defined in handled.

Also, for drivers, the `--quiet` and `--verbose` flags are added at the task level (e.g. `uw upp run`), not at the mode level (e.g. `uw upp`), so we will also not have that information when `--show-schema` is specified. And since it does't really make sense in this context anyway, we can default to `False` for both.